SU-SWS/decanter
# added semibold and other source sans pro font styles that were missing

# READY FOR REVIEW

# Summary
- Added missing Source Sans Pro font styles
- Address gaps between mocks and code output in projects

# Needed By (Date)
- N/A

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that the new font styles (font-style and font-weight) are available (e.g., font-weight: 900)

# Affected Projects or Products
- Decanter
- Archaeology

# Associated Issues and/or People
- N/A

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
